src/main/java/pl/byczazagroda/trackexpensesappbackend/dto/CreateWalletDTO.java

```diff
@@ -0,0 +1,9 @@
+package pl.byczazagroda.trackexpensesappbackend.dto;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+import java.time.Instant;
+
+public record CreateWalletDTO(@NotBlank @Size(max = 20) @Pattern(regexp = "[a-z A-Z]+") String name, Instant creationTime) {
```

Comment:
Why you add time here? Frontend shouldn't send us creation time. It should be done in backend when wallet will be save in db.